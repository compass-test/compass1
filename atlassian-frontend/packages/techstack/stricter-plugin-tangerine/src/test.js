const { rules } = require('./index');

describe('Stricter-plugin-tangerine', () => {
  it('should export the correct rules', () => {
    expect(rules).toMatchSnapshot();
  });
});
