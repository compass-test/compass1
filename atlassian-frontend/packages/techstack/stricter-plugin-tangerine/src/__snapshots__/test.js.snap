// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stricter-plugin-tangerine should export the correct rules 1`] = `
Object {
  "dummy": Object {
    "onProject": [Function],
  },
  "dummy-fail": Object {
    "onProject": [Function],
  },
  "project-structure": Object {
    "dirsMatch": [Function],
    "filesMatch": [Function],
    "matchHierarchy": [Function],
    "meta": Object {
      "docs": Object {
        "description": "Enforce a specific file structure for your project",
      },
    },
    "onProject": [Function],
    "performMatch": [Function],
    "prepareStruct": [Function],
    "processMismatches": [Function],
    "resolveDefinitionAlias": [Function],
  },
}
`;
